luigi/contrib/spark.py

```diff
@@ -292,6 +292,10 @@ def files(self):
         if self.deploy_mode == "cluster":
             return [self.run_pickle]
 
+    @property
+    def pickle_protocol(self):
+        return configuration.get_config().getint(self.spark_version, "pickle-protocol", pickle.DEFAULT_PROTOCOL)
```

Comment:
why is this pulling from `self.spark_version` config section rather than the `spark` config section (`py-packages` appears to pull from a config section called `spark`)

Reply:
Thanks for your review. Updated it just like `py-packages`.